src/coord/src/coord.rs

```diff
@@ -510,13 +510,19 @@ impl Coordinator {
                         .load_timestamp_bindings(entry.id())
                         .expect("loading timestamps from coordinator cannot fail");
 
+                    let persist_desc = self
+                        .persister
+                        .load_source_persist_desc(&source)
+                        .map_err(CoordError::Persistence)?;
+
```

Comment:
Did you mean to keep this? Seems like you intended to re-use the persist desc from above?

Reply:
I mean to delete it, pretty sure. It turns out a persistence description is not timeless; it has things like current compaction frontiers and stuff, so it becomes outdated pretty much immediately, and isn't a valuable part of `create_sources`.

Reply:
Deleted!